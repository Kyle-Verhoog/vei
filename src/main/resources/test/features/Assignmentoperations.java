// FAIL
// Java 1.3
public class Assignmentoperations {
  public void m(int x) {
    x+=42;
    x-=42;
    x*=42;
    x/=42;
    x%=42;
    x|=42;
    x&=42;
    x^=42;
    x<<=42;
    x>>=42;
    x>>>=42;
  }
}