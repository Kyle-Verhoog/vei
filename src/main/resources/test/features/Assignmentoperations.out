PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
VOID ┃ void
IDENTIFIER ┃ m
( ┃ (
INT ┃ int
IDENTIFIER ┃ x
) ┃ )
{ ┃ {
IDENTIFIER ┃ x
+ ┃ +
= ┃ =
INTEGER_LITERAL ┃ 42
; ┃ ;
IDENTIFIER ┃ x
- ┃ -
= ┃ =
INTEGER_LITERAL ┃ 42
; ┃ ;
IDENTIFIER ┃ x
* ┃ *
= ┃ =
INTEGER_LITERAL ┃ 42
; ┃ ;
IDENTIFIER ┃ x
/ ┃ /
= ┃ =
INTEGER_LITERAL ┃ 42
; ┃ ;
IDENTIFIER ┃ x
% ┃ %
= ┃ =
INTEGER_LITERAL ┃ 42
; ┃ ;
IDENTIFIER ┃ x
| ┃ |
= ┃ =
INTEGER_LITERAL ┃ 42
; ┃ ;
IDENTIFIER ┃ x
& ┃ &
= ┃ =
INTEGER_LITERAL ┃ 42
; ┃ ;
IDENTIFIER ┃ x
^ ┃ ^
= ┃ =
INTEGER_LITERAL ┃ 42
; ┃ ;
IDENTIFIER ┃ x
< ┃ <
<= ┃ <=
INTEGER_LITERAL ┃ 42
; ┃ ;
IDENTIFIER ┃ x
> ┃ >
>= ┃ >=
INTEGER_LITERAL ┃ 42
; ┃ ;
IDENTIFIER ┃ x
> ┃ >
> ┃ >
>= ┃ >=
INTEGER_LITERAL ┃ 42
; ┃ ;
} ┃ }
} ┃ }